server/services/conditionService.ts

```diff
@@ -1,34 +1,34 @@
 import LicenceApiClient from '../data/licenceApiClient'
-import RegionOfResidence from '../routes/creatingLicences/types/additionalConditionInputs/regionOfResidence'
-import RestrictionOfResidency from '../routes/creatingLicences/types/additionalConditionInputs/restrictionOfResidency'
-import MedicalAppointmentType from '../routes/creatingLicences/types/additionalConditionInputs/medicalAppointmentType'
-import UnsupervisedContact from '../routes/creatingLicences/types/additionalConditionInputs/unsupervisedContact'
-import WorkingWithChildren from '../routes/creatingLicences/types/additionalConditionInputs/workingWithChildren'
-import IntimateRelationshipWithGender from '../routes/creatingLicences/types/additionalConditionInputs/intimateRelationshipWithGender'
-import BehaviourProblems from '../routes/creatingLicences/types/additionalConditionInputs/behaviourProblems'
-import AppointmentTimeAndPlace from '../routes/creatingLicences/types/additionalConditionInputs/appointmentTimeAndPlace'
-import OutOfBoundsRegion from '../routes/creatingLicences/types/additionalConditionInputs/outOfBoundsRegion'
-import OutOfBoundsPremises from '../routes/creatingLicences/types/additionalConditionInputs/outOfBoundsPremises'
-import OutOfBoundsPremisesType from '../routes/creatingLicences/types/additionalConditionInputs/outOfBoundsPremisesType'
-import DrugTestLocation from '../routes/creatingLicences/types/additionalConditionInputs/drugTestLocation'
-import ElectronicMonitoringTypes from '../routes/creatingLicences/types/additionalConditionInputs/electronicMonitoringTypes'
-import ElectronicMonitoringPeriod from '../routes/creatingLicences/types/additionalConditionInputs/electronicMonitoringPeriod'
-import ApprovedAddress from '../routes/creatingLicences/types/additionalConditionInputs/approvedAddress'
-import AlcoholMonitoringPeriod from '../routes/creatingLicences/types/additionalConditionInputs/alcoholMonitoringPeriod'
-import AlcoholRestrictionPeriod from '../routes/creatingLicences/types/additionalConditionInputs/alcoholRestrictionPeriod'
-import CurfewTerms from '../routes/creatingLicences/types/additionalConditionInputs/curfewTerms'
-import CurfewAddress from '../routes/creatingLicences/types/additionalConditionInputs/curfewAddress'
-import NoContactWithVictim from '../routes/creatingLicences/types/additionalConditionInputs/noContactWithVictim'
-import ReportToApprovedPremises from '../routes/creatingLicences/types/additionalConditionInputs/reportToApprovedPremises'
+import RegionOfResidence from '../routes/manageConditions/types/additionalConditionInputs/regionOfResidence'
```

Comment:
These service layer modules referencing route level modules are a bit whiffy.. violate direction of dependencies, they should probably be pulled into the service layer